docs/src/pages/patterns/Header/Header.example.jsx

```diff
@@ -1,15 +1,7 @@
-import { Choice, ChoiceList, Header } from '@design-system';
+import { ChoiceList, Header } from '@design-system';
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-const variationChoices = [
```

Comment:
Updated the examples so we can see all variations side by side and not have to toggle between them.